### ISSUE TITLE
Dandere2xPR v0.0.6 Feedback

### ISSUE THREAD

**Issue by DCGYT** [akai-katto/dandere2x-python-rework#18]
Hello again, there has been a breakthrough in the interface.

Regarding the bugs issue, they are still the same. Having to use "Quality Coefficent" at maximum to avoid white dots, "Block Size" high and "Tile Size" at minimum to avoid lines.

Since version 0.0.4.0 I had noticed that the appearance of the videos had changed, now they were more grainy and not as smooth as previous versions. I did not know what to think about it, I thought that maybe it was a bug, but it remains in version 0.0.6. Unless it's a setting to change the quality that I haven't seen.

I don't really understand what "Model" does, I have compared the 3 models above in DaVinci Resolve (re-encoded x264 with avidemux) with a difference filter and they are all exactly the same, should they change something?

If the graininess issue is going to stay in future versions, I would like there to be an option to make it smooth like it was until version 0.0.3.0.

3.7.2 Legacy [vulkan][s2][n3][b60][q100]
![image](https://user-images.githubusercontent.com/98280315/211167928-a06c82a4-060d-4d9b-9006-5b02bcf82bdc.png)
0.0.3.0 [q100*][b60][d3][cunet][t050][w2]
![image](https://user-images.githubusercontent.com/98280315/211167800-b5e8efd8-edc8-444a-ac29-730f67d068f9.png)
0.0.6 [q10][b60][s2][p1][d3][cunet][t050][w2]
![image](https://user-images.githubusercontent.com/98280315/211167820-31944857-1f16-45e4-8c1a-6a1e62c797cf.png)

https://drive.google.com/drive/folders/1ZxyjDErgJGih6EyGzMY_QBjTN2gwtMLV?usp=share_link

**Comment by akai-katto** [comment]
Thanks for pointing out the graininess issue, I know where it probably came from. I'll work on a fix. 

Again, much appreciated.

**Comment by akai-katto** [comment]
I found the source of the behavior. Expect a fix out tonight.

**Comment by akai-katto** [comment]
https://github.com/akai-katto/dandere2x-python-rework/releases/tag/0.0.6.1

**Comment by DCGYT** [comment]
@akai-katto The issue was fixed, thank you very much, but still the same issues with noise and everything else.

Is there any documentation on everything yaml does because I don't understand it much, or if more experimental settings can be added to try to track down where the errors are coming from?

**Comment by akai-katto** [comment]
Can you be more specific with your request? 

1) "Quality coefficient" refers to how "alike" two blocks can be for them to be considered a "valid block match".

2) "Tile Size" is the setting that [waifu2x-ncnn-vulkan](https://github.com/nihui/waifu2x-ncnn-vulkan) uses in order to "tile across" the image. Larger means it'll sample more pixels, causing the distortion that you notice, while smaller is slower, but will cause less distortion. 

Do you have any other questions? I'd  be happy to answer, sorry if I haven't been the most direct.

**Comment by DCGYT** [comment]
@akai-katto I am referring to the "yaml", which I don't understand very well how it works. I would like to see if by modifying some parameters I can reduce the bugs. Also if there is a possibility to add more experimental parameters to see if it somehow changes something to the video. Because I haven't seen that it has been modified in the last versions.

On the subject of distortion, in my opinion,  I see that it is the same only in a smaller grid, the noise is always the same.

**Comment by akai-katto** [comment]
@DCGYT are you referring to `output_options.yaml`?

**Comment by DCGYT** [comment]
@akai-katto Yes, I would like the noise issue to be completely solved, because when applying sharpening filters it is too noticeable.
